### DISCUSSION TITLE
functionality in Safari : swipe between pages to navigate

### DISCUSSION THREAD

**Discussion by ffuuzzyyllooggiicc** [pilotmoon/Scroll-Reverser#175] [General]
Hello. On installing Scroll Reverser I noticed a change in the behaviour where the swipe gestures in Safari for navigating between pages no longer work. Is there a fix?

**Comment by pilotmoon** [comment]
Turn of "reverse horizontal" setting. Flip system natural scroll setting and other settings to gatin desired behavour

--
Nick Moore
Pilotmoon Software
https://pilotmoon.com/

**Reply by ffuuzzyyllooggiicc** [reply]
Many thanks!
